src/config/mod.rs

```diff
@@ -78,6 +78,10 @@ pub fn balances_cache_duration() -> usize {
     env_with_default("BALANCES_REQUEST_CACHE_DURATION", 60 * 1000)
 }
 
+pub fn balances_core_cache_duration() -> usize {
+    env_with_default("BALANCES_CORE_REQUEST_CACHE_DURATION", indefinite_timeout())
```

Comment:
I would match the method name to the env var name. Also add entry in the test in `config/tests/mod.rs` where we assert default values and env var names.

As an optional comment: I would suggest renaming to `balance_request_cache_duration` and the env var `BALANCE_REQUEST_CACHE_DURATION`

Reply:
> I would match the method name to the env var name. Also add entry in the test in config/tests/mod.rs where we assert default values and env var names.

Changed in 527a259

> As an optional comment: I would suggest renaming to balance_request_cache_duration and the env var BALANCE_REQUEST_CACHE_DURATION

There's already a `BALANCES_REQUEST_CACHE_DURATION` which is for the CGW route. That's why I create a new one with `BALANCES_CORE_REQUEST_CACHE_DURATION`